CMakeModules/FindPythonQt.cmake

```diff
@@ -24,15 +24,15 @@ string(REGEX REPLACE
 )
 
 if(NOT EXISTS "${PYTHONQT_INSTALL_DIR}")
-  find_path(PYTHONQT_INSTALL_DIR include/PythonQt/PythonQt.h DOC "Directory where PythonQt was installed.")
+  find_path(PYTHONQT_INSTALL_DIR include/PythonQt5/PythonQt.h DOC "Directory where PythonQt was installed.")
```

Comment:
One issue here is that you're replacing what works for existing releases with the Debian 9 path; it would be better to **add** an extra name to search for (here, using `NAMES`).